internal/backends/qt/qt_window.rs

```diff
@@ -1349,6 +1349,13 @@ impl PlatformWindow for QtWindow {
         }
     }
 
+    fn set_rendering_notifier(
+        &self,
+        _callback: Box<dyn sixtyfps_corelib::window::RenderingNotifier>,
+    ) {
+        panic!("set_rendering_notifier() is not supported in the Qt backend.")
```

Comment:
Is it better to panic or to ignore? or maybe print a warning?
Is there a way to detect it from the application whether or not it's going to be supported?
Perhaps one should return an Error

Reply:
That's a very good question. The the very least the function should indeed return an error if it fails, i.e. if the backend doesn't support it - or if it is called twice then. What would be the right type? `std::result::Result<(), Box<dyn std::error::Error>`?

Reply:
Maybe `std::result::Result<(), SetRenderingNotifierError>`

```rust 
enum SetRenderingNotifierError { Unsupported, AlreadySet }
```

Something like that.